File: classes/parte-6-datos-y-estructuras/089-arreglos-de-tamano-fijo/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	f := strings.Fields(line)
	var arr [3]int
	for i := 0; i < 3; i++ {
		arr[i], _ = strconv.Atoi(f[i])
	}
	suma, max := 0, arr[0]
	for _, x := range arr {
		suma += x
		if x > max {
			max = x
		}
	}
	fmt.Printf("suma=%d max=%d\n", suma, max)
}

File: classes/parte-6-datos-y-estructuras/089-arreglos-de-tamano-fijo/implementaciones/c/main.c
#include <stdio.h>

int main(void) {
    long arr[3];
    if (scanf("%ld %ld %ld", &arr[0], &arr[1], &arr[2]) != 3) return 1;
    long suma = 0, max = arr[0];
    for (int i = 0; i < 3; i++) {
        suma += arr[i];
        if (arr[i] > max) max = arr[i];
    }
    printf("suma=%ld max=%ld\n", suma, max);
    return 0;
}
